client/state/selectors/is-site-store.js

```diff
@@ -0,0 +1,24 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { isJetpackSite } from 'state/sites/selectors';
+/**
+ * Returns true if site is a Automated Transfer site, false if not and null if unknown
```

Comment:
This comment doesn't seem quite right - its not guaranteeing this is an AT site, but only a Jetpack site with WooCommerce, right?